src/applications/claims-status/components/RequestedFilesInfo.jsx

```diff
@@ -28,7 +28,7 @@ export default class RequestedFilesInfo extends React.Component {
               <div className="item-container">
                 <h3 className="file-request-title">{item.displayName}</h3>
                 <p className="submission-description">
-                  {truncateDescription(item.description)}
+                  {stripHtml(truncateDescription(item.description))}
```

Comment:
not sure how much over the truncation limit these would get, but would it make more sense to flip the nesting and truncate _after_ stripping?

Reply:
💯